pkg/client/src/app/pages/applications/components/application-list-expanded-area/application-list-expanded-area-analysis.tsx

```diff
@@ -93,7 +92,7 @@ export const ApplicationListExpandedAreaAnalysis: React.FC<
             <Tooltip content="Click to view Analysis report">
               <Button variant="link" isInline>
                 <Link
-                  to={`/hub/applications/${application.id}/bucket${task?.data?.output}/`}
+                  to={`/hub/applications/${application.id}/bucket${task?.data?.output}`}
```

Comment:
No relevant to this PR and the slash is needed because windup drops in the bucket an index.html.

Reply:
It is not needed - this PR fixes the 401 when linking to this api call. The index.html is not part of the url path - raw html is served from the api - The url looks like this 

http://192.168.49.2/hub/applications/1/bucket/windup/report/